<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addgraph</class>
 <widget class="QWidget" name="addgraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QQLabel {
    qproperty-alignment: AlignCenter; /* توسيط النص (بدون CSS عادي!) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                               stop:0 #3498db, stop:1 #2ecc71); /* جرادينت */
    border: 2px solid #2980b9;
    border-radius: 10px;
    padding: 10px;
text-align: center;  
}
QLabel:hover {
    color: #ffffff;
    background-color: rgba( 86, 101, 115, 0.5); 
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>اسم الخريطه</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_warning">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;       /* Border color */
     
    padding: 5px;
    margin: 0;                    /* Remove default margin */
}

/* Last QLineEdit should have a right border */
QLineEdit:last-child {
    border-right: 1px solid #ccc; 
}</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:  #C1FFC1;
		border:none;
		border-radius: 3px;
		text-align: left;
		padding: 8px 0 8px 15px;
		color: #788596;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, &quot;Helvetica Neue&quot;, sans-serif;
border-radius: 10px;

	}
QPushButton:hover {
		background-color: rgba( 86, 101, 115, 0.5);
	}
 QPushButton:checked {
		color: #fff;
	}</string>
       </property>
       <property name="text">
        <string>اضف اسم</string>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/icons/add-row-16.ico</normaloff>:/icons/add-row-16.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>11</width>
         <height>11</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QQLabel {
    qproperty-alignment: AlignCenter; /* توسيط النص (بدون CSS عادي!) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                               stop:0 #3498db, stop:1 #2ecc71); /* جرادينت */
    border: 2px solid #2980b9;
    border-radius: 10px;
    padding: 10px;
text-align: center;  
}
QLabel:hover {
    color: #ffffff;
    background-color: rgba( 86, 101, 115, 0.5); 
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>من فين؟!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_waring2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;       /* Border color */
     
    padding: 5px;
    margin: 0;                    /* Remove default margin */
}

/* Last QLineEdit should have a right border */
QLineEdit:last-child {
    border-right: 1px solid #ccc; 
}</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QQLabel {
    qproperty-alignment: AlignCenter; /* توسيط النص (بدون CSS عادي!) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                               stop:0 #3498db, stop:1 #2ecc71); /* جرادينت */
    border: 2px solid #2980b9;
    border-radius: 10px;
    padding: 10px;
text-align: center;  
}
QLabel:hover {
    color: #ffffff;
    background-color: rgba( 86, 101, 115, 0.5); 
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>لفين؟!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_waring3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QQLabel {
    qproperty-alignment: AlignCenter; /* توسيط النص (بدون CSS عادي!) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                               stop:0 #3498db, stop:1 #2ecc71); /* جرادينت */
    border: 2px solid #2980b9;
    border-radius: 10px;
    padding: 10px;
text-align: center;  
}
QLabel:hover {
    color: #ffffff;
    background-color: rgba( 86, 101, 115, 0.5); 
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>هتدفع كام ؟!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_waring4">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:  #C1FFC1;
		border:none;
		border-radius: 3px;
		text-align: left;
		padding: 8px 0 8px 15px;
		color: #788596;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, &quot;Helvetica Neue&quot;, sans-serif;
border-radius: 10px;

	}
QPushButton:hover {
		background-color: rgba( 86, 101, 115, 0.5);
	}
 QPushButton:checked {
		color: #fff;
	}</string>
       </property>
       <property name="text">
        <string>اضافه</string>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/icons/add-16.ico</normaloff>:/icons/add-16.ico</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
